content/docs/user-guide/components/reference/vegetation/vegetation-area-system.md

```diff
@@ -0,0 +1,73 @@
+---
+linkTitle: Vegetation Area System
+title: Vegetation Area System Component
+description: The Vegetation Area System component handles the placement and removal of vegetation instances based on its configured rules in Open 3D Engine (O3DE).
+---
+
+The **Vegetation Area System** component handles the placement and removal of vegetation instances throughout your world, based on its configured rules.
+
+Learn about how the Vegetation Area System component works within the vegetation system in [Technical details](#technical-details) below.
+
+{{< note >}}
+This is a system component that exists when you interact with the vegetation system via the Vegetation Gem. Generally, this system is already set up and you don't need to do anything further to manage vegetation in your world.
```

Comment:
The properties of the Area System and Instance System components seem more like defaults or project defaults, all of these properties are configurable per-level with the _level component_, Vegetation System Settings.  You might consider amending this note to point that out, and possibly link to docs for those components when they are added.

Reply:
Updated this, and updated it for Vegetation Instance system component. Pages for those don't exist yet so I didn't add a link.

Reply:
Also thank you, great call out, this is helpful to know!